### ISSUE TITLE
Installation doc changes

### ISSUE THREAD

**Issue by henn** [CCI-MOC/hil#502]
Based on feedback from @jfontecchio, we need to update:
1. The section on `haas.cfg`, to say that we no longer need to symlink in /var/... and talk about `HAAS_ENDPOINT`
2. The need to copy haas.cfg to /var/www/haas

**Comment by jfontecchio** [comment]
I am going through the install again to see what else I missed.  When installing the packages required I noticed that python-virtinst is no longer available in the public repository for CentOS 7.  Was the python-virtinst package deprecated and replaced with virt-manager?

**Comment by jfontecchio** [comment]
You might want to include git as package that is a prerequisite as you need it later for the install?

**Comment by jfontecchio** [comment]
There needs to be some clarity when to run commands as root and when to run commands as haas_user during the install.

**Comment by jfontecchio** [comment]
You should consider adding a comment in the SELinux section about how to put it in permissive mode even after reboots.

[root@haas haas]# cat /etc/selinux/config 

# This file controls the state of SELinux on the system.
# SELINUX= can take one of these three values:
#     enforcing - SELinux security policy is enforced.
#     permissive - SELinux prints warnings instead of enforcing.
#     disabled - No SELinux policy is loaded.
SELINUX=permissive
# SELINUXTYPE= can take one of three two values:
#     targeted - Targeted processes are protected,
#     minimum - Modification of targeted policy. Only selected processes are protected. 
#     mls - Multi Level Security protection.
SELINUXTYPE=targeted

**Comment by jfontecchio** [comment]
Is gcc a requirement?  When building HaaS I get the following error below.

Writing /tmp/easy_install-Jc0zx9/coverage-4.0.1/setup.cfg
Running coverage-4.0.1/setup.py -q bdist_egg --dist-dir /tmp/easy_install-Jc0zx9/coverage-4.0.1/egg-dist-tmp-gcklLn
warning: no previously-included files found matching 'ci/appveyor.token'
no previously-included directories found matching 'doc/_build'
no previously-included directories found matching 'tests/eggsrc/build'
no previously-included directories found matching 'tests/eggsrc/dist'
no previously-included directories found matching 'tests/eggsrc/*.egg-info'
warning: no previously-included files matching '*.py[co]' found anywhere in distribution
unable to execute gcc: No such file or directory
**
** Couldn't install with extension module, trying without it...
** BuildFailed: command 'gcc' failed with exit status 1
**
warning: no previously-included files found matching 'ci/appveyor.token'
no previously-included directories found matching 'doc/_build'
no previously-included directories found matching 'tests/eggsrc/build'
no previously-included directories found matching 'tests/eggsrc/dist'
no previously-included directories found matching 'tests/eggsrc/*.egg-info'
warning: no previously-included files matching '*.py[co]' found anywhere in distribution

**Comment by jfontecchio** [comment]
Under the "Running the Server under Apache" section there was one place I got tripped up and took me awhile to debug.  I thought I had everything working but I kept getting a connection refused when trying to run HaaS commands.  I made sure SELinux was disabled and the firewall.  I made sure that the service was listening on the correct port.  I however copied the wsgi file into /var/www/haas as root.  The file needs to be owned as haas_user.  That would be a nice little note to add to that section as I over looked it in the configuration where it says the service will be running as haas_user.  It was a newbie mistake but could save people some time and frustration in the future.
